colors/material.vim

```diff
@@ -414,5 +414,26 @@ if has('nvim')
   let g:terminal_color_13 = g:terminal_color_5
   let g:terminal_color_14 = g:terminal_color_6
   let g:terminal_color_15 = g:terminal_color_7
+elseif has('terminal')
```

Comment:
Do you mind explaining why this is necessary?

Reply:
I actually researched this a bit and didn't find it necessary 😄

Although I saw `g:terminal_ansi_colors` defined in many colorschemes.

I can remove it.

Reply:
Sounds good! I can merge this as soon as that's fixed up. Thanks!

Reply:
done